### PR TITLE
#162727463 Implement gmail as EAAS provider

### PR DESCRIPTION
**Pull request by wombolo**
#### What does this PR do?

In order to have scalabilty at production environment the email API provider has been switched from elasticemail to gmail.

#### Description of Task to be completed?
Fix Email API implementation

#### How should this be manually tested?
Register a new user In production environment

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162727463

#### Questions:

### CODE REVIEW

**Review by julietezekwe — APPROVED** [review]
Looks good except for the dropped coverage

**Comment by abejide001** [comment]
Looks good to me

**Review by Proception — COMMENTED** [review]
Dropped coverage, is there a valid reason for this?

**Review by abejide001 — APPROVED** [review]
LGTM

File: src/helpers/EmailNotificationAPI.js [hunk 1]
```diff
@@ -27,15 +27,11 @@ class EmailNotificationAPI {
    */
   static transportCreator() {
     const {
-      EMAILHOST, EMAILPORT, EMAILUSER, EMAILPASS
+      EMAILUSER, EMAILPASS
     } = process.env;
-    if (!EMAILUSER || !EMAILPASS || !EMAILHOST) {
```
Comment (andela-dbamidele) [inline src/helpers/EmailNotificationAPI.js:32]: Why did you remove this check?

Comment (wombolo) [reply src/helpers/EmailNotificationAPI.js:32]: There was no coverage on this line because they are environment variables and such will always be set by us

Comment (andela-dbamidele) [reply src/helpers/EmailNotificationAPI.js:32]: But what happens when I forgot to set them?

Comment (wombolo) [reply src/helpers/EmailNotificationAPI.js:32]: The app will crash
I'll revert and try to cover the line
